tests/damage_rheology_friction_heating/statistics

```diff
@@ -0,0 +1,29 @@
+# 1: Time step number
+# 2: Time (years)
+# 3: Time step size (years)
+# 4: Number of mesh cells
+# 5: Number of Stokes degrees of freedom
+# 6: Number of temperature degrees of freedom
+# 7: Number of degrees of freedom for all compositions
+# 8: Iterations for temperature solver
+# 9: Iterations for composition solver 1
+# 10: Iterations for Stokes solver
+# 11: Velocity iterations in Stokes preconditioner
+# 12: Schur complement iterations in Stokes preconditioner
+# 13: Minimal value for composition olivine_grain_size
+# 14: Maximal value for composition olivine_grain_size
+# 15: Global mass for composition olivine_grain_size
+# 16: Minimal temperature (K)
+# 17: Average temperature (K)
+# 18: Maximal temperature (K)
+# 19: Average nondimensional temperature (K)
+# 20: RMS velocity (m/year)
+# 21: Max. velocity (m/year)
+# 22: Total viscous dissipation (J/yr)
+# 23: Average shear heating rate (W/kg) 
+# 24: Total shear heating rate (W) 
+0 0.000000000000e+00 0.000000000000e+00 256 2467 1089 1089 0  0 30 31 92 1.00000000e-03 1.00000010e-03 1.00000005e+07 1.60000000e+03 1.60000000e+03 1.60000000e+03 3.73686192e-15 5.77350322e-01 9.92956151e-01 1.64416789e+10 1.70092552e-11 5.78331687e+02 
+1 3.125000000000e+03 3.125000000000e+03 256 2467 1089 1089 9 14  2  3  9 9.98808684e-04 9.98808927e-04 9.98808867e+06 1.60000000e+03 1.60000132e+03 1.60000152e+03 5.54208165e-07 5.77350417e-01 9.92956243e-01 1.63845356e+10 1.69503369e-11 5.76328384e+02 
+2 6.250000000000e+03 3.125000000000e+03 256 2467 1089 1089 6  8  1  2  6 9.97659878e-04 9.97660195e-04 9.97660129e+06 1.60000000e+03 1.60000261e+03 1.60000300e+03 1.10115573e-06 5.77350604e-01 9.92956364e-01 1.63295237e+10 1.68936134e-11 5.74399709e+02 
+3 9.375000000000e+03 3.125000000000e+03 256 2467 1089 1089 6  8  1  2  6 9.96544556e-04 9.96544921e-04 9.96544857e+06 1.60000000e+03 1.60000390e+03 1.60000445e+03 1.64236848e-06 5.77350845e-01 9.92956492e-01 1.62762052e+10 1.68386333e-11 5.72530311e+02 
+4 1.000000000000e+04 6.250000000000e+02 256 2467 1089 1089 3  6  0  0  0 9.96306624e-04 9.96307004e-04 9.96306933e+06 1.60000000e+03 1.60000415e+03 1.60000473e+03 1.74996623e-06 5.77350893e-01 9.92956518e-01 1.62648483e+10 1.68269221e-11 5.72132118e+02 
```

Comment:
The original results are 9.96306619e-04 9.96306991e-04 9.96306927e+06 (for the min, max, mass of grain size), again very similar to the new results, so this test is fine.